src/sentry/tsdb/redis.py

```diff
@@ -195,7 +195,19 @@ def get_model_key(self, key):
             # enforce utf-8 encoding
             if isinstance(key, six.text_type):
                 key = key.encode("utf-8")
-            return md5(repr(key)).hexdigest()
+
+            key_repr = repr(key)
+            if six.PY3:
+                # XXX: Python 3 reprs bytes differently to Python 2. In py2,
```

Comment:
Shall we add one of those TODO comments, instructing us to remove this after the move?

Reply:
We can't ever remove this logic, but yeah we could remove the `if six.PY3` branch so that we always do it. Will add.